tests/platform_tests/test_platform_info.py

```diff
@@ -228,7 +228,8 @@ def check_all_psu_on(dut, psu_test_results):
         cli_psu_status = dut.command(CMD_PLATFORM_PSUSTATUS_JSON)
         psu_info_list = json.loads(cli_psu_status["stdout"])
         for psu_info in psu_info_list:
-            psu_test_results[psu_info['name']] = psu_info
+            if psu_info["status"] != 'NOT PRESENT':
+                psu_test_results[psu_info['name']] = psu_info
```

Comment:
Should this check be added for 201911/201811 os_version also, to be used on non-chassis platforms as well?

Reply:
It'll be a good-to-have.
right now only chassis have the PSUs `NOT PRESENT` as the test was working for T0/T1. I checked on some T0 and T1 multi-asic, they don't have this.
```
PSU    Status
-----  --------
PSU 1  OK
PSU 2  OK
```